content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -346,6 +346,36 @@ service chaining and flexible load balancing.
 There are multiple, flexible ways to install JuniperContrail/TungstenFabric CNI.
 
 Kindly refer to this quickstart: [TungstenFabric](https://tungstenfabric.github.io/website/)
+{{% /tab %}}
+{{% tab name="Cilium" %}}
+Cilium is open source software for providing and transparently securing network connectivity and loadbalancing between application workloads.
+
+Passing `--pod-network-cidr` option to `kubeadm init` is not required, but highly recommended.
+
+These commands will deploy Cilium with its own etcd managed by etcd operator.
+
+```shell
+# download required manifests from cilium repo
+wget https://github.com/cilium/cilium/archive/v1.2.0.zip
+unzip v1.2.0.zip
+cd cilium-1.2.0/examples/kubernetes/addons/etcd-operator
+
+# generate and deploy etcd certificates
+export CLUSTER_DOMAIN=$(kubectl get ConfigMap --namespace kube-system coredns -o yaml | awk '/kubernetes/ {print $2}')
+tls/certs/gen-cert.sh $CLUSTER_DOMAIN
+tls/deploy-certs.sh
```

Comment:
Do these scripts need to run with root privilege?

Reply:
No, only `tls/deploy-certs.sh` needs `kubectl` to be configured.